src/Program.cs

```diff
@@ -9,23 +9,9 @@
 
 namespace b2c_ms_graph
 {
-    class Program
+    public static class Program
     {
-        static void Main(string[] args)
-        {
-            try
-            {
-                RunAsync().GetAwaiter().GetResult();
-            }
-            catch (Exception ex)
-            {
-                Console.ForegroundColor = ConsoleColor.Red;
-                Console.WriteLine(ex.Message);
-                Console.ResetColor();
-            }
-        }
-
-        private static async Task RunAsync()
+        static async Task Main(string[] args)
```

Comment:
Entry point in console app can be "async Task", so no need to `RunAsync().GetAwaiter().GetResult()`